elm.js

```diff
@@ -0,0 +1,5312 @@
+(function(scope){
```

Comment:
consider git-ignoring this compiled file. 😉

Reply:
Yes I was going to add it to gitignore but I need it in the `index.html` file to be able to run the application (hosted with Github pages:  https://dwyl.github.io/elm-pwa-example/). The example for a `Browser.application` app is working now I'm going to update the code to use our API to capture text